public/video-ui/src/components/YoutubeFurniture/index.js

```diff
@@ -46,6 +47,26 @@ class YoutubeFurniture extends React.Component {
       : null;
   };
 
+  composerKeywordsToYouTube = () => {
```

Comment:
This copies across the tags into the YouTube tab, following some rules about replacement etc